packages/manager/modules/pci/src/components/project/instance/instance.constants.js

```diff
@@ -5,6 +5,17 @@ export const CAPABILITIES = {
   FAIL_OVER_IP: 'failoverip',
 };
 
+/* TODO:
```

Comment:
Still TODO ?

Reply:
Yes, i respond on another comment i think.
I created a Jira task for that.
I will remove it once i made the changes.